.editorconfig

```diff
@@ -0,0 +1,22 @@
+# See https://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_size = 2
+indent_style = space
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.json]
+indent_style = tab
+
```

Comment:
I'd be inclined to just use spaces; we're planning to switch to spaces for json in cursorless too

```suggestion
```